packages/webapp/public/locales/en/translation.json

```diff
@@ -23,12 +23,12 @@
     "COMPOSITION": "Composition",
     "COMPOSITION_ERROR": "Error: The total percentage of N, P, K, and additional nutrients must not exceed 100%. Please adjust your values.",
     "COPPER": "Copper (Cu)",
-    "DRY_FERTILISER": "Dry fertiliser",
+    "DRY_FERTILISER": "Dry",
     "DRY_MATTER_CONTENT": "Dry matter content",
     "EDIT_PRODUCT_DETAILS": "Edit product details",
-    "FERTILISER_TYPE": "Fertiliser type",
-    "FERTILISER_TYPE_PLACEHOLDER": "What type of fertiliser is it?",
-    "LIQUID_FERTILISER": "Liquid fertiliser",
+    "FERTILISER_TYPE": "Is this product dry or liquid?",
+    "FERTILISER_TYPE_PLACEHOLDER": "Select...",
```

Comment:
This is the default so you can just delete the key entirely and also the prop on the React Select which refers to it.